style.css

```diff
@@ -1,648 +1,1324 @@
-body {
-  display: block;
-  margin: 0%;
-}
+@media only screen and (min-width: 768px) {
+  body {
+    display: block;
```

Comment:
- [ ] ❤️, You have respected the media query width as precise the requirement but am sorry to say that it is applying to the page yet, when am on this screen size or wider, the design is broken, I would recommend you fix all up before you resubmit your work @edahigure, please take a look at this below


![5](https://user-images.githubusercontent.com/77843050/206590030-297503bd-3294-47d5-bcb7-3b61df8bd3da.gif)

- [ ] Please watch this again and again this record above to detect all the places I highlight with my `cursor`, and consider making the fix on the `mobile version` too, your web app needs to be perfect on both sides, this is the reason why I have started to record the mobile-first. please take a look back at this [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=dJfKIXmZHIUSnAFQ-0) template and correct yours